pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
-  version = "2.11.0";
+  version = "2.14.1";
```

Comment:
I think we should do [2.14.2](https://github.com/helm/helm/releases/tag/v2.14.2)